cli/src/format.rs

```diff
@@ -312,16 +312,31 @@ fn response_to_table<T>(resp: &JobStatusResponse<T>) -> Table
 where
     T: Scored,
 {
-    let ecosystem = PackageType::from_str(&resp.ecosystem).unwrap_or(PackageType::Npm);
-    let ecosystem_label = match ecosystem {
-        PackageType::Npm => "NPM",
-        PackageType::RubyGems => "RubyGems",
-        PackageType::PyPi => "PyPI",
-        PackageType::Maven => "Maven",
-        PackageType::Nuget => "NuGet",
-        PackageType::Golang => "Golang",
-        PackageType::Cargo => "Cargo",
-    };
+    // Combine all ecosystems into a comma-separated string.
+    let ecosystems = resp
+        .ecosystems
+        .iter()
+        .flat_map(|ecosystem| {
+            Some(match PackageType::from_str(&ecosystem).ok()? {
+                PackageType::Npm => "NPM",
+                PackageType::RubyGems => "RubyGems",
+                PackageType::PyPi => "PyPI",
+                PackageType::Maven => "Maven",
+                PackageType::Nuget => "NuGet",
+                PackageType::Golang => "Golang",
+                PackageType::Cargo => "Cargo",
+            })
+        })
+        .collect::<Vec<_>>();
+    let mut ecosystems_value = ecosystems.join(", ");
+
+    // Add fallback if no ecosystem could be identified.
```

Comment:
This fallback path was encountered in a situation where it wasn't expected:

<img width="1086" alt="image" src="https://user-images.githubusercontent.com/18729796/214905837-f20d5229-16c1-44b5-827c-c6f36c378be2.png">

---

The same `analyze` command performed with CLI v4.4.0 shows the ecosystem as `Cargo` whereas with the code from this PR it shows as `Unknown`.

Reply:
This turned out to be due to testing on PROD instead of STG.